entries/entry01.md

```diff
@@ -1,7 +1,21 @@
 # Entry 1
-##### X/X/XX
+##### 10/20/19
 
-Text
+The tool I considered for this project was the Arduino kit and Makey Makey set.  
+
+I decided to use the Arduino kit can be used to deal with voltage and switches which is something that my idea needs in order to function properly.
+I chose the Arduino kit over the Makey Makey set because more professionals use the Arduino kit to deal with real life projects. 
+I want to create ***SmartGlass***. SmartGlass is a sort of technology that is not very popular amongst individuals.  
```

Comment:
In my brief research, this looks expensive. Do you have a budget for the materials? The school can definitely supply an Arduino for testing. Have you thought about the rest of the materials? If you propose a budget, the school could possibly fund it.